frontend/src/App.tsx

```diff
@@ -1,21 +1,32 @@
 import {useEffect, useState} from "react";
 import axios from "axios"
+import {CharacterCard} from "./CharacterCard.ts";
 
 export default function App() {
 
-
     const [playingCards, setPlayingCards] = useState<string[]>([""]);
+    const [characterCards, setCharacterCards] = useState<CharacterCard[]>([]);
     const gameSize = "large";
 
     function loadPlayingCards() {
-         axios.get<string[]>(
+        axios.get<string[]>(
             "api/playing_cards/" + gameSize)
             .then((response) => {
-                    setPlayingCards(response.data)
-                })
-             .catch(console.error)
+                setPlayingCards(response.data)
+            })
+            .catch(console.error)
     }
 
+    function loadCharacterCards() {
```

Comment:
consider to refactor your fetching functions to a seperate file. If you are familiar with custom hooks or global state management, try to do that instead.

Reply:
🙏